apps/ui/src/helpers/connectors/sequence-waas.ts

```diff
@@ -0,0 +1,48 @@
+import Connector from './connector';
+
+const projectAccessKey = 'AQAAAAAAAKD5WUrmbwO_EbPcAST_dV5PpDg';
```

Comment:
Sorry you had the right key actually previously, I though I deleted my comment
```suggestion
const projectAccessKey = 'AQAAAAAAAJbd_5JOcE50AqglZCtvu51YlGI';
```

Reply:
The access key you are suggesting now is my own that I created on my private account 🤔

Reply:
Humm weird I doubled checked and I got the same in my account, maybe its not a production key

Reply:
Should I change it back to this?

Reply:
Yes, the other one that I mentioned is not the right key for Sequence connector